src/app/footer-nav/contact-us.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { Location } from '@angular/common';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'ccd-contact-us',
+  templateUrl: './contact-us.component.html',
+  styleUrls: ['./footer-nav.component.scss']
+})
+export class ContactUsComponent implements OnInit {
+
+  constructor(public router: Router,
+              private route: ActivatedRoute,
+              private location: Location) { }
+
+  ngOnInit() {
+  }
+
```

Comment:
Question Do we need OnInit and ngOnInit?

Reply:
Removed them